src/crypto/utils/constants.py

```diff
@@ -15,4 +15,6 @@
 DATASET = config("DATASET", cast=str)
 DAY_DIM = f"{PROJECT_ID}.{DATASET}.{config('DAY_DIM', cast=str)}"
 MONTH_DIM = f"{PROJECT_ID}.{DATASET}.{config('MONTH_DIM', cast=str)}"
+DATE_DIM = f"{PROJECT_ID}.{DATASET}.{config('DATE_DIM', cast=str)}"
```

Comment:
is that python-decouple? then I think str is already the default, no?

Reply:
yes, it is. You're right. I removed the use of cast.